Content.Shared/SubFloor/SubFloorHideComponent.cs

```diff
@@ -10,17 +14,43 @@ namespace Content.Shared.SubFloor
     /// (plating).
     /// </summary>
     /// <seealso cref="P:Content.Shared.Maps.ContentTileDefinition.IsSubFloor" />
+    [NetworkedComponent]
     [RegisterComponent]
     public sealed class SubFloorHideComponent : Component
     {
         /// <inheritdoc />
         public override string Name => "SubFloorHide";
 
         /// <summary>
-        ///     This entity needs to be anchored to be hid in the subfloor.
+        ///     Whether the entity will be hid when not in subfloor.
+        /// </summary>
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("enabled")]
+        public bool Enabled { get; set; } = true;
```

Comment:
Man if only every component had a `Running` property that could be used to toggle them on and of...

Reply:
`Running` has changed, old man.
![image](https://user-images.githubusercontent.com/6766154/127771420-e73df758-9d5e-45b5-8ea8-58a9cedfaa33.png)